.evergreen.yml

```diff
@@ -42,24 +45,26 @@ functions:
       type: setup
       params:
         working_dir: src/github.com/mongodb/mongodb-enterprise-kubernetes/tools/multicluster
-        add_to_path:
-          - src/github.com/mongodb/tools
-          - src/github.com/mongodb/tools/linux_amd64
         include_expansions_in_env:
           - GITHUB_TOKEN
           - macos_notary_keyid
           - macos_notary_secret
           - workdir
         env:
+          <<: *go_env
           MACOS_NOTARY_KEY: ${macos_notary_keyid}
           MACOS_NOTARY_SECRET: ${macos_notary_secret}
+          GORELEASER_CURRENT_TAG:
+        # shell.exec EVG Task doesn't have add_to_path, so we need to explicitly add the path export below.
         script: |
           set -Eeu pipefail
-          
-          ${workdir}/goreleaser release
+
+          export PATH=$GOROOT/bin:$PATH
+          ${workdir}/goreleaser release --rm-dist
 
 tasks:
   - name: package_goreleaser
+    git_tag_only: true
```

Comment:
according to our evergreen setting page and this documentation: https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings/#triggering-versions-with-git-tags this should suffice. 

`mms-build-account` is part of the allowed users to create tags. So we will have the following release process:
- mms-build-account creates a new release tag
- evergreen picks that up and will run this tasks/variants
- profit? 💸